### DISCUSSION TITLE
Error from server (InternalError): error when creating "experiment.yaml": Internal error occurred: failed calling webhook "vauth.kb.io": Post https://chaos-mesh-controller-manager.chaos-testing.svc:443/validate-auth?timeout=5s: x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by fuyuteng** [chaos-mesh/chaos-mesh#3695] [Q&A]
第一次普通安装用的是install.sh方式，提示dockerapi版本问题，后来用helm install chaos-mesh chaos-mesh/chaos-mesh   -n chaos-testing --set chaosDaemon.env.DOCKER_API_VERSION=1.39 --set controllerManager.leaderElection.enabled=false  --set dashboard.securityMode=false --version 2.3.2，但是我的实验删除不掉

Error from server (InternalError): error when creating "experiment.yaml": Internal error occurred: failed calling webhook "[vauth.kb.io](http://vauth.kb.io/)": Post [https://chaos-mesh-controller-manager.chaos-testing.svc:443/validate-auth?timeout=5s](https://chaos-mesh-controller-manager.chaos-testing.svc/validate-auth?timeout=5s): x509: certificate signed by unknown authority

**Comment by cwen0** [comment]
@fuyuteng You can refer this docs to force delete the experiments https://chaos-mesh.org/docs/run-a-chaos-experiment/#delete-chaos-experiments-using-commands

**Reply by fuyuteng** [reply]
不行强制删除不了，我只能删除crd,就是httpchaos crd的才可以，但是这样实验就不行

**Reply by fuyuteng** [reply]
但是我就算把这个删除了，我创建实现就报证书错误
